### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by gavindidrichsen**
Hi Russell, Stuart.  The only outstanding work on the README is to get the powershell commands added and a couple asciicinema examples of the command-line added.  I'll raise another PR to get these added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,15 +30,19 @@ Frontend and Backend will be configured with individual Availability Sets and Pr
...
+- a local azure client should be installed, either the CLI or Powershell client variations.  To install the the Azure CLI see [these instructions](https://docs.microsoft.com/en-gb/cli/azure/install-azure-cli?view=azure-cli-latest) or to install the Azure Powershell client see [these instructions](https://docs.microsoft.com/en-us/powershell/azure/install-azurerm-ps?view=azurermps-6.3.0)
+- The AD identity running this installation should have the **Owner** role on the required Subscription.
+- jq should be installed to allow easy parsing of JSON output.  See [installation instructions here](https://stedolan.github.io/jq/download/)
 
 ## Installation Instructions
 
-### 1. Create Service Principal [optional]
+### 1. Ensure a valid Service Principal exists
+
+The template shares information like private keys and passwords between the servers with a Key Vault Resource.  Using an existing service principle credential the template deployment process creates the key vault.  Only a process (or user) using this same service principle can read or write to this key vault.
```
Comment (skpaterson) [inline README.md]: Would be good to update each `principle` -> `principal`

File: README.md [hunk 2]
```diff
@@ -68,43 +72,61 @@ stuart@Azure:~$ az ad sp show --id a530c3a0-YOUR-GUID-HERE-21e3d7ede80c
     }
 ```
 
-Note the values for **appId**, **objectId** and **password** for the parameters file.
+__Using the Powershell:__ (TBD)
```
Comment (skpaterson) [inline README.md]: Why is this TBD?

**Review by skpaterson — APPROVED** [review]
Couple of small comments.  Good work!

**Comment by gavindidrichsen** [comment]
@skpaterson.  Thanks.  I've updated the README as discussed and according to your comments.  Cheers.

**Review by skpaterson — APPROVED** [review]
👍

**Comment by gavindidrichsen** [comment]
Hi @skpaterson, @russellseymour.  Can you review my latest powershell updates to the README so that this PR can be merged into trunk?  These are all the powershell equivalents to the CLI commands added to the README

**Review by russellseymour — APPROVED** [review]
This looks great.

When you add scripts for adding nodes etc make sure you add the PS equivalents as well.

**Pull request merged**